src/main/java/carpet/script/annotation/Locator.java

```diff
@@ -140,7 +141,7 @@ private Locators() {}
 
         private static class BlockLocator<R> extends AbstractLocator<R>
         {
-            private final boolean returnBlockValue;
+            private final java.util.function.Function<BlockArgument, R> returnFunction;
```

Comment:
any reason can't import java.util.function.Function?

Reply:
`Locator.Function`, since it technically is just `Function`. Not sure if I really tried to manually import though, but I think it didn't let me.

Maybe I could give it its own `FunctionalInterface` to make it easier to look at.